bridge/engine/ActiveMediaList.cpp

```diff
@@ -521,6 +517,8 @@ void ActiveMediaList::process(const uint64_t timestamp,
     size_t speakerCount = rankSpeakers();
     if (speakerCount == 0)
     {
+        TActiveTalkersSnapshot activeTalkersSnapshot;
```

Comment:
If we do not set here we will have a list with at least one active talker though all are muted.